Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -2154,11 +2154,9 @@ void webkit_settings_set_javascript_can_open_windows_automatically(WebKitSetting
  */
 gboolean webkit_settings_get_enable_hyperlink_auditing(WebKitSettings* settings)
 {
-    g_return_val_if_fail(WEBKIT_IS_SETTINGS(settings), FALSE);
-
-    g_warning("webkit_settings_get_enable_hyperlink_auditing is deprecated and always returns FALSE.");
```

Comment:
If this is not desired you should also remove it from webkit_settings_get_enable_dns_prefetching. Or is it still desired if it returns false?

Reply:
I was thinking about just deleting all of the warnings, because they're not very useful.

But I think they are OK as long as we don't warn when setting to default value, which happens during WebKitSettings construction as every one of these setters gets called.